client/src/index.css

```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Harmattan&family=Tajawal&display=swap');
```

Comment:
if you wonder like me, what is font-display [this link will help you](https://css-tricks.com/almanac/properties/f/font-display/)